app/Providers/AuthServiceProvider.php

```diff
@@ -54,7 +54,7 @@ public function boot()
         Passport::tokensCan([
             'friends.read' => trans('api.scopes.friends.read'),
             'identify' => trans('api.scopes.identify'),
-            'users.read' => trans('api.scopes.users.read'),
```

Comment:
there are quite a few active `users.read` token at the moment. Are we going to force them to reauthorize?

Although I also noticed the scope check is currently an `all` (as in requires all the specified scopes) instead of `any` (requires any of the specified scopes)

Reply:
yes, the grant is different